cypress/e2e/tests/migration/applicationinventory/analysis/source_analysis.test.ts

```diff
@@ -347,6 +347,25 @@ describe(["@tier1"], "Source Analysis", () => {
         application.verifyEffort(this.analysisData["tackle-testapp-public-4-targets"]["effort"]);
     });
 
+    // Automates customer bug MTA-2973
+    it(["@tier0"], "Source analysis on tackle app public with custom rule", function () {
+        const application = new Analysis(
+            getRandomApplicationData("tackle-public-customRule", {
+                sourceData: this.appData["tackle-testapp-public"],
+            }),
+            getRandomAnalysisData(this.analysisData["tackle-testapp-public-customRule"])
+        );
+        cy.wait(2 * SEC);
+        Application.open();
+        application.create();
+        application.manageCredentials(null, maven_credential.name);
+        applicationsList.push(application);
+        cy.wait(5 * SEC);
+        application.analyze();
+        application.verifyAnalysisStatus("Completed");
+        application.verifyEffort(this.analysisData["tackle-testapp-public-customRule"]["effort"]);
```

Comment:
In the TC we need to compare results of two analysis : 
1) source analysis with maven setting file 
2) source analysis without maven setting file 
Results should be same

Reply:
Thanks, Done @sshveta